task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   name: clamav-scan
 spec:
   description: >-
-    Scans image for malware with ClamAV.
+    Scans content of container images for viruses, malware, and other malicious content using ClamAV antivirus scanner.
```

Comment:
Suggestion: Scan the content of container images for viruses, malware, and other malicious content with the ClamAV antivirus scanner.

Reply:
![image](https://user-images.githubusercontent.com/90042568/228845639-43029a82-9f97-4746-860f-7b98be8426c8.png)

Reply:
Thanks! I'd prefer to go with "using" because it puts more weight on the tool used. I'll update this everywhere.